### PR TITLE
Merge to Master for v1.0.11

### PR DESCRIPTION
**Pull request by mjvcallibrity**
Establish a dependency on Crossroads.Web.Common v1.1.15

### CODE REVIEW

File: Crossroads.ClientApiKeys.Test/src/DomainLockedApiKeyFilterIntegrationTest.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using Microsoft.Practices.Unity;
 using Microsoft.Practices.Unity.Configuration;
 using NUnit.Framework;
+using Unity;
```
Comment (jpc11623) [inline Crossroads.ClientApiKeys.Test/src/DomainLockedApiKeyFilterIntegrationTest.cs]: Is Unity necessary here? I'm not seeing any other changes to the file.

File: Crossroads.ClientApiKeys/App_Start/UnityConfig.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+using Unity;
+using Unity.WebApi;
+
+namespace Crossroads.ClientApiKeys
+{
+    public static class UnityConfig
+    {
+        public static void RegisterComponents()
+        {
+			var container = new UnityContainer();
```
Comment (jpc11623) [inline Crossroads.ClientApiKeys/App_Start/UnityConfig.cs]: Indentation

File: Crossroads.ClientApiKeys.Test/src/DomainLockedApiKeyFilterIntegrationTest.cs (hunk 1 above)
Comment (mjvcallibrity) [reply Crossroads.ClientApiKeys.Test/src/DomainLockedApiKeyFilterIntegrationTest.cs]: Yes. It is brings in access to UnityContainer.

File: Crossroads.ClientApiKeys/App_Start/UnityConfig.cs (hunk 2 above)
Comment (mjvcallibrity) [reply Crossroads.ClientApiKeys/App_Start/UnityConfig.cs]: Not sure why the indentation is displaying that way in github. When I view this file in Visual Studio, it appears to be indented in a manner consistent with the rest of the project.

**Pull request merged**